ansible/README.md

```diff
@@ -60,9 +62,51 @@ By default, if you omit the `-i` parameter, the `local` environment will be used
 
 In all instructions, replace `<openwhisk_home>` with the base directory of your Openwhisk source tree. e.g. `openwhisk`
 
+#### Setup
+
+This step needs to be done only once per development environment. It will generate configuration files based on your local settings. Notice that for the following playbook you don't need to specify a target environment as it will run only local actions.
+After the playbook is done you should see a file called `db_local.ini` in your ansible directory. It will by default contain settings for a local ephemeral CouchDB setup. Afterwards, you can change the values directly in `db_local.ini` 
+
+#####  Ephemeral CouchDB
+
+If you want to use the ephemeral CouchDB, run this command
+
+```
+ansible-playbook setup.yml
```

Comment:
Should add a tip that if that fails with this error:

`ERROR! Unexpected Exception: (setuptools 1.1.6 (/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python), Requirement.parse('setuptools>=11.3'))`

Then: `pip install --upgrade setuptools --user python`